src/Azure/Azure.Quantum.Client/JobManagement/IWorkspace.cs

```diff
@@ -47,16 +73,24 @@ public interface IWorkspace
         /// Lists the jobs.
         /// </summary>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>List of jobs</returns>
-        Task<IEnumerable<CloudJob>> ListJobsAsync(
+        /// <returns>List of jobs.</returns>
+        IAsyncEnumerable<CloudJob> ListJobsAsync(
```

Comment:
Ah, that is much nicer than having the entire enumerable being the result of an async task!